static/operator-profile-minimal.yaml

```diff
@@ -5,6 +5,5 @@ metadata:
   namespace: istio-operator
   name: example-istiocontrolplane
 spec:
-  profile: default
```

Comment:
Is this correct? shouldn't it be minimal?

Reply:
the automated tooling does some seding here, but it appears I didn't revert enough PRs - will address.